go-controller/pkg/node/controllers/upgrade/node_upgrade.go

```diff
@@ -14,11 +15,12 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/selection"
-	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/klog/v2"
 )
 
+var ErrStopped = errors.New("network stopped")
```

Comment:
`failed to retrieve topology version, network stopped` to be more similar to the error returned on timeout.

Or perhaps the timeout error should be just `timeout`, depending on what the caller is appending to the error.